### ISSUE TITLE
Is there any example which performs regression using L2 loss? 

### ISSUE THREAD

**Issue by prashant905** [ppwwyyxx/tensorpack#189]
I want to perform regression on resnet-Dorefanet. I have to manipulate last with L2 loss . But inside the models,  i don't see any code for L2 loss. Can i still do it, by using the standard tensorflow notations?

**Comment by prashant905** [comment]
Also, I am simply trying to train resnet-dorefa.py with this command `./resnet-dorefa.py --dorefa 1,4,6 --data dorefa/ --gpu 0` .

Inside dorefa folder i have two folders train and val. train folder has small chunk of imagenet dataset. it has just two folders n02115913  and n02116738.  and similarly dorefa has val folder which have folder n02115913  and n02116738 containing annotations of train folders.

The command just gets executed without any output or errors.

**Comment by ppwwyyxx** [comment]
So what is the issue?

**Comment by ppwwyyxx** [comment]
"Look at the docstring in *-dorefa.py to see detailed usage and performance."

**Comment by prashant905** [comment]
My question is after running this command `./resnet-dorefa.py --data dorefa/ --gpu 0` on a CPU just to check the code with a small dataset of imagenet ILSVRC12 (like 15mb). It just run and finish without any output , so i am confused whether it should print something or show me the training status on the console. 
Also , where does resent-dorefa.py it stores the npy files if it trains successfully?

**Comment by ppwwyyxx** [comment]
···
"""
This script loads the pre-trained ResNet-18 model with (W,A,G) = (1,4,32)
It has 59.2% top-1 and 81.5% top-5 validation error on ILSVRC12 validation set.
To run on images:
    ./resnet-dorefa.py --load pretrained.npy --run a.jpg b.jpg
To eval on ILSVRC validation set:
    ./resnet-dorefa.py --load pretrained.npy --eval --data /path/to/ILSVRC
"""
···
These are the only two usages of this script. It is not a training script.
